0.pre-preprocess_images/scripts/change_file_names.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Replace the strings to rename the file names
+
+# In[ ]:
+
+
+import pathlib
+
+# In[ ]:
+
+
+# set path to the input files
+input_path = pathlib.Path("../../data/00.raw_images_ims/").resolve()
+
+# set path to the output files
+output_path = pathlib.Path("../../data/0.raw_images_tiff/").resolve()
+
+# create output path if it does not exist
+output_path.mkdir(parents=True, exist_ok=True)
+
+# get all files in the input path
+files = input_path.glob("*.ims")
+
+
+# In[ ]:
+
+
```

Comment:
Maybe it might be worth in the code comment adding what the file will be left with. This might also can be included in the README about what is left in the file names that are important. Just a suggestion.